### ISSUE TITLE
Starting DBeaver leads to error because of a offline DB connection that DBeaver tries to auto-connect at startup

### ISSUE THREAD

**Issue by AntonTrapp** [dbeaver/dbeaver#11242]
#### System information: 
- Windows 10 20H2 (build 19042.746)
- 7.3.4.2002101311042

#### Connection specification:
- Postgres 13
- PostgreSQL JDBC Driver
- no

#### Describe the problem you're observing:
Opening DBeaver it always tries to connect to one database that is offline. This feature can not be turned off.

Turning off SQL Editor in the generic settings (from my search even though no editor is open at all) does not help in any way.

#### Steps to reproduce, if exist:
1) Open DBeaver after having defined a database connection when this specific DB is not running
2) Error that it can not connect appearing

#### Include any warning/errors/backtraces from the logs
none

**Comment by kseniiaguzeeva** [comment]
Disable **Connect on editor activation** in SQL Editor settings. It should help.
![dbeaver_CUFwecUdZ5](https://user-images.githubusercontent.com/51405061/107614882-382a0100-6c5c-11eb-920c-c6b0a5d60779.png)

**Comment by AntonTrapp** [comment]
Thank you for your answer, but unfortunately no, that was the first I googled.

Can it be that changing this setting does not do anything if the connection can not be opened?

I tried to disable that feature later on while I had an active connection and removed / added the connection again. Can't tell what did the trick, but just disabling the option while the DB has not been accessible didn't work. If the change-when-not-active stuff is a reproducable bug I would put it on the todo list, if not I would close the issue as I got a workaround and nobody should work only for my convenience 😄

**Comment by kseniiaguzeeva** [comment]
Do you have any object opened in Data Editor when you launch DBeaver? If yes, please try to disable Restore editor state on startup (in Preferences>User interface>Editors) or close objects before exiting DBeaver.
![Screenshot 2021-02-18 at 10 55 38](https://user-images.githubusercontent.com/51405061/108323658-f9ee8d80-71d7-11eb-9f71-e34a80141431.png)

**Comment by AntonTrapp** [comment]
Thank you for your answer. Yes, this setting seems to be activated by default. It still is (I disabled it now). But like described above I was able to get rid of the problem by starting dbeaver with a connection to the 1 problematic DB and disconnected/reconnected the DB again and the problem was gone. It does not seem to happen any more (and never happened for any other connection - most of my connections are ssh tunnels and offline almost all of the time). So it seems like some strange thing that has been happening once.
